instrumentation/tests.bzl

```diff
@@ -0,0 +1,9 @@
+"""
+Contains all the end-to-end test classes under the instrumentation directory.
```

Comment:
Out of curiosity, why do we need this?

Reply:
This was a previous implementation.
Removed now.